solutionbox/code_free_ml/test_mltoolbox/test_analyze_data.py

```diff
@@ -715,7 +716,7 @@ def _open_and_encode_image(img_url):
         img = Image.open(f).convert('RGB')
         output = cStringIO.StringIO()
         img.save(output, 'jpeg')
-      return output.getvalue()
+      return base64.urlsafe_b64encode(output.getvalue())
```

Comment:
You are right. I did not include 2 files that I should have (they are in different dir)!

Added.